### PR TITLE
Feature/query parents

### LINKED ISSUE #887: need contextual information on showpage about parent items
when a work has a parent work, the child work's showpage should have some contextual information pointing to that parent work. 

any recollection as to why we haven't done that?

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #887 

This hacks in a VERY not hyraxy fix for bringing the parent work info to the show page of a title. it simply shows the parent titles for a child.

### CODE REVIEW

**Comment by straleyb** [comment]
screen shot 2018-04-25 at 2 57 58 pm

**Comment by straleyb** [comment]
screen shot 2018-04-25 at 3 02 01 pm

File: app/views/hyrax/base/_relationships_parent_rows.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<%# Render presenters which aren't specified in the 'presenter_types' %>
+<% presenter.grouped_presenters(except: presenter.presenter_types).each_pair do |model_name, items| %>
+  <%= render 'relationships_parent_row', type: model_name, items: items, presenter: presenter %>
+<% end %>
+
+<dt> <%= "Parents" %> </dt>
```
Comment (straleyb) [inline app/views/hyrax/base/_relationships_parent_rows.html.erb]: I'll extract this into its own partial to keep the code isolated. But I don't expect it to change shape much. Trying to keep this as basic as possible.

Comment (revgum) [inline app/views/hyrax/base/_relationships_parent_rows.html.erb]: Move to i18n

File: app/views/hyrax/base/_relationships_parent_rows.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<dd>
+    <% if ScholarsArchive::ParentQueryService.query_parents_for_id(presenter.solr_document.id).blank? %>
+      <p><%= "Empty" %></p>
```
Comment (revgum) [inline app/views/hyrax/base/_relationships_parent_rows.html.erb]: Move to i18n with text like 'This work has no parent.'

File: config/locales/hyrax.en.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 en:
+  parents: "Parents"
```
Comment (revgum) [inline config/locales/hyrax.en.yml]: lets try to keep with the hyrax way.. this belongs somewhere under
```
hyrax:
  base:
    relationships_parent_rows_label: "Parents:"
```

Parents should have the trailing semi-colon so it looks like the admin set labe.

File: app/services/scholars_archive/parent_query_service.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+module ScholarsArchive
+  class ParentQueryService
+    def self.query_parents_for_id(child_id)
```
Comment (revgum) [inline app/services/scholars_archive/parent_query_service.rb]: Can you write a test for this service and just mock the returns to guard against this changing unintentionally

File: config/locales/hyrax.en.yml (hunk 3 above)
Comment (revgum) [reply config/locales/hyrax.en.yml]: also.. add a corresponding `.........rows_empty_text: "This work has no parents."`

**Comment by straleyb** [comment]
Outdated after updates from master. Closed for new PR.

**Pull request closed without merging**